frontend/src/app/(routes)/(overview)/overview-components/PageAd.tsx

```diff
@@ -0,0 +1,12 @@
+import Image from "next/image";
+import React from "react";
+
+const PageAd = () => {
+  return (
+    <div className="my-10 cursor-pointer brightness-50">
+      <Image className="w-full" src="/page-ad-sample.png" width={1000} height={80} alt="Ad" />
```

Comment:
```suggestion
      <Image className="w-full" src="https://dummyimage.com/100X80/000/fff&text=Overview+Ad1" width={1000} height={80} alt="Ad" />
```

You can use this URL to render images with arbitrary width and hight.